change/@fluentui-babel-preset-global-context-c4a6b25c-8ca3-4466-8b10-43aadd3b0941.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
This should be patch to trigger the release 🚨

Reply:
we dont have broken releases - that's why I used none, any upcoming v9 release  will be without issues no ?